AxonDeepSeg/__init__.py

```diff
@@ -1,7 +1,15 @@
 import os
 import sys
+import pygit2
```

Comment:
Welp, it looks like the error was _not_ related to `GitPython`: 

```
  ModuleNotFoundError: No module named 'pygit2'
```

It seems like it instead has something to do with the build process of the `axondeepseg` package, and the fact that we have a non-stdlib import in our `__init__.py`. 

I'll keep debugging!

Reply:
Well, in that case maybe we could just move all of the git version stuff out of init and into ads_utils.py and then import from there - that should be a quick fix?

Reply:
Okay, I think the reason this is failing is because in `setup.py`, we import `AxonDeepSeg.__version__`, and thus `import pygit2`/`import GitPython` gets hit when installing our package in editable mode.

While looking up this pattern, I found an [old post](https://stackoverflow.com/a/2058872) with a little edit at the bottom describing something similar to our dilemma:

> UPDATE (2017): coverage.py no longer imports itself to get the version. Importing your own code can make it uninstallable, because you product code will try to import dependencies, which aren't installed yet, because setup.py is what installs them.

But, the dependencies _should_ be installed? They're separate lines! Weird.

Anyway, I think maybe we should fix this by not importing from `AxonDeepSeg` to get the version. I'll give it another shot...

Reply:
>Okay, I think the reason this is failing is because in setup.py, we import AxonDeepSeg.__version__, and thus import pygit2/import GitPython gets hit when installing our package in editable mode

Ahh I missed that!

>But, the dependencies should be installed? They're separate lines! Weird.

ikr! installed but maybe the env/PATH isn't "seen" yet during setup for some reason?

Reply:
It looks like the import of `AxonDeepSeg` into `setup.py` was indeed the problem! :tada:

Reply:
Fantastic! Thank you so much!